torch/ao/quantization/quantizer/utils.py

```diff
@@ -3,74 +3,9 @@
 import torch
 from torch.ao.quantization.quantizer.quantizer import (
     QuantizationAnnotation,
-    QuantizationConfig,
-    QuantizationSpec,
 )
 from torch.fx import Node
 
-__all__ = [
-    "get_input_act_qspec",
-    "get_output_act_qspec",
-    "get_weight_qspec",
-    "get_bias_qspec",
-]
-
-
-def get_input_act_qspec(quantization_config: Optional[QuantizationConfig]):
```

Comment:
Same for this. Moving to quantizer_utils. Although if we are not yet aligned on the visiblity of these concepts, then keeping them in xnnpack_quantizer_utils is fine.

Reply:
yeah I think we can put this in xnnpack_quantizer_utils for now, see my comment in https://github.com/pytorch/pytorch/pull/106642#discussion_r1287813709